### PR TITLE
Implement On-Premises page

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**

### CODE REVIEW

File: src/components/PricingPage/ComparePlans/Columns.jsx [hunk 1]
```diff
@@ -48,12 +48,13 @@ export const Columns = ({ title = '', cols }) => {
       {title && <div className={getCompareContainer('__row-title')}>{title}</div>}
       {isVisibility && (
         <div className={getCompareContainer('__row-title-cols')}>
-          {cols.map((item, index) => (
+          {cols.map((columnValue, index) => (
             <div
-              key={constructElementKey(item, index)}
+              key={constructElementKey(columnValue, index)}
               className={cx(getCompareContainer('__row-title-col'))}
+              data-short={mobileColumns[columnValue] ?? columnValue}
```
Comment (OleksandrDragun) [inline src/components/PricingPage/ComparePlans/Columns.jsx:55]: Why do we need this attribute?

File: src/components/PricingPage/TrustedOrganizations/TrustedOrganizations.scss [hunk 2]
```diff
@@ -3,14 +3,14 @@
 
 .trusted-organizations {
   &__container {
-    margin-top: 117px;
+    margin-top: 130px;
 
-    @include m.breakpoint(v.$tablet-sm-exact) {
-      margin-top: 111px;
+    @include m.breakpoint(v.$tablet-sm) {
```
Comment (OleksandrDragun) [inline src/components/PricingPage/TrustedOrganizations/TrustedOrganizations.scss]: I think you need to review all scss changes and switch v.$tablet-sm to v.$tablet-sm-exact according to https://github.com/reportportal/reportportal.github.io/tree/new-landing#scsscss
Please let me know if I'm wrong.

File: src/components/IconBlock/IconBlock.scss [hunk 3]
```diff
@@ -28,8 +35,14 @@
...
   height: 291px;
-  background: url('icons/PentagonSubtract.svg') no-repeat -8px -2px;
+  background: no-repeat -8px -2px;
   background-size: 105%;
+
+  @for $i from 1 through 5 {
+    &-#{$i} {
+      background-image: url('icons/Pentagon#{$i}.svg');
+    }
+  }
```
Comment (ViktorSoroka07) [inline src/components/IconBlock/IconBlock.scss:45]: Nice😎

File: src/components/PricingPage/TrustedOrganizations/TrustedOrganizations.scss (hunk 2 above)
Comment (ViktorSoroka07) [reply src/components/PricingPage/TrustedOrganizations/TrustedOrganizations.scss]: I agree, also noticed this

File: src/components/PricingPage/ComparePlans/Columns.jsx (hunk 1 above)
Comment (yelyzavetakhokhlova) [reply src/components/PricingPage/ComparePlans/Columns.jsx:55]: I'm using them to display shortened column names for mobile via css in order to not use media react-responsive as much as possible
image

**Review by ViktorSoroka07 — DISMISSED** [review]

File: src/components/PricingPage/ComparePlans/ComparePlans.scss [hunk 4]
```diff
@@ -128,6 +128,12 @@
       }
     }
 
+    .ant-collapse-item:last-child {
+      @include m.breakpoint(v.$tablet-lg) {
```
Comment (OleksandrDragun) [inline src/components/PricingPage/ComparePlans/ComparePlans.scss]: $tablet-sm-exact
In all this file.

Comment (yelyzavetakhokhlova) [reply src/components/PricingPage/ComparePlans/ComparePlans.scss]: Updated everywhere

**Review by ViktorSoroka07 — DISMISSED** [review]

**Review by OleksandrDragun — DISMISSED** [review]

**Review by ViktorSoroka07 — APPROVED** [review]

**Pull request merged**